### PR TITLE
Add: magic-wormhole 0.12.0

### PR DESCRIPTION
**Pull request by aquacash5**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/79685)

### CODE REVIEW

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS0.CRLF.7-2-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: magic-wormhole.magic-wormhole
```
Comment (Trenly) [inline manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.installer.yaml:4]:
```suggestion
PackageIdentifier: aquacash5.magic-wormhole
```
For releases through github, its best to use the Owner.Package format for package identifiers

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS0.CRLF.7-2-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: magic-wormhole.magic-wormhole
```
Comment (Trenly) [inline manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: aquacash5.magic-wormhole
```
For releases through github, its best to use the Owner.Package format for package identifiers

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS0.CRLF.7-2-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: magic-wormhole.magic-wormhole
```
Comment (Trenly) [inline manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.yaml:4]:
```suggestion
PackageIdentifier: aquacash5.magic-wormhole
```
For releases through github, its best to use the Owner.Package format for package identifiers

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ShortDescription: Get things from one computer to another, safely.
+# Description:
+Moniker: wormhole
+Tags:
+  - file-share
+  - magic
+  - python
+  - wormhole
+# Agreements:
+ReleaseNotes: |
```
Comment (Trenly) [inline manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml:27]:
```suggestion
ReleaseNotes: |-
```

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS0.CRLF.7-2-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: magic-wormhole.magic-wormhole
+PackageVersion: 0.12.0
+PackageLocale: en-US
+Publisher: magic-wormhole
```
Comment (Trenly) [inline manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml:7]:
```suggestion
Publisher: aquacash5
```
Best to have this be the repo owner

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.installer.yaml (hunk 1 above)
Comment (aquacash5) [reply manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.installer.yaml:4]: The package I am uploading is just a repackaging I made of the project for Chocolatey. The real owner of the project is correctly attributed https://github.com/magic-wormhole/magic-wormhole

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml (hunk 2 above)
Comment (aquacash5) [reply manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml:4]: The package I am uploading is just a repackaging I made of the project for Chocolatey. The real owner of the project is correctly attributed https://github.com/magic-wormhole/magic-wormhole

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.yaml (hunk 3 above)
Comment (aquacash5) [reply manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.yaml:4]: The package I am uploading is just a repackaging I made of the project for Chocolatey. The real owner of the project is correctly attributed https://github.com/magic-wormhole/magic-wormhole

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml (hunk 5 above)
Comment (aquacash5) [reply manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml:7]: The package I am uploading is just a repackaging I made of the project for Chocolatey. The real owner of the project is correctly attributed https://github.com/magic-wormhole/magic-wormhole

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+PackageUrl: https://magic-wormhole.readthedocs.io/en/latest/
+License: MIT
+LicenseUrl: https://github.com/magic-wormhole/magic-wormhole/blob/master/LICENSE
+Copyright: Copyright (c) 2015 Brian Warner
+# CopyrightUrl:
+ShortDescription: Get things from one computer to another, safely.
+# Description:
+Moniker: wormhole
+Tags:
+  - file-sharess
```
Comment (ItzLevvie) [inline manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml:22]:
```suggestion
  - file-shares
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml (hunk 6 above)
Comment (aquacash5) [reply manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml]: I'm going to change the tag to `file-sharing`. That seems to be the correct tag.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**